README.md

```diff
@@ -1 +1,26 @@
-# BookListApp-Microverse-Project
\ No newline at end of file
+# BookListApp-Microverse-Project
+
+## Description
+This is a basic website that allows users to add/remove books from a list. This project will be achieved using using JavaScript objects and arrays as well as, dynamically serving data to the DOM and basic events.
+
+
+<img width="348" alt="awesome_books_basic_ui" src="https://user-images.githubusercontent.com/48631109/148910731-10077881-7fbc-4278-bd9a-8ecc2960c411.png">
+
+## Getting started
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
+
+* Once the files are on your machine, open the html-css-javascript-getting-started folder in Visual Studio Code.
+
+* With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with Live Server.
+
+## Technologies Used
+* CSS
+* HTML
+* GitHub
+
```

Comment:
- [ ] You did great improving your README file, But there are a few sections you might have missed, Kindly add the acknowledgment,authors, licence,.. sections to your README file to make it look more professional, Use [this Microverse template](https://github.com/microverseinc/readme-template)

![image](https://user-images.githubusercontent.com/75126481/148912578-92eaf3af-ed63-459f-b6eb-107e5c215ed3.png)